shared/src/main/scala/org/atnos/eff/FutureEffect.scala

```diff
@@ -24,6 +24,15 @@ trait FutureCreation {
 
   def async[R, A](a: =>A)(implicit m: Member[Future, R], ec: ExecutionContext): Eff[R, A] =
     send(Future(a))
+
+  def lift[R, A](
```

Comment:
Finally it would be nice to have a bit of syntax in `syntax.future` to be able to write `myFuture.liftFuture`.